localflavor/bg/fields.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+from .validators import egn_validator, eik_validator
+
+
+class EGNField(models.CharField):
```

Comment:
This means that the field's should be called `BGEGNField` and `BGEIKField` right?